sig-aws/charter.md

```diff
@@ -0,0 +1,98 @@
+# SIG AWS Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG AWS is responsible for the creation and maintenance of subprojects (features/innovations) necessary to integrate AWS services for the operation and management of Kubernetes on AWS. SIG AWS also acts as a forum for Kubernetes on AWS users/developers to raise their feature requests and support issues with. SIG leads in collaboration with SIG members will make a best effort to triage known problems within one or two release cycle of issues being reported. SIG AWS in collaboration with SIG-Testing, SIG-Scalability and SIG-Docs is responsible for integration and maintenance of tests (e2e, periodic jobs, postsubmit jobs etc.); scale-tests (load, density tests) and documentation for the scope within the purview of this charter.
+
+### In scope
+
+Link to SIG [subprojects](https://github.com/kubernetes/community/tree/master/sig-aws#subprojects)
+
+#### Code, Binaries and Services
+
+Kubernetes integrations specific to AWS including:
+- Integrations, interfaces, libraries and extension points for all AWS services such as IAM, storage, networking, loadbalancers, registry, security, monitoring/logging at the instance or container level
+- Tools for Kubernetes APIs to work with AWS services including Amazon EKS
+- Prow, testgrid, perf dashboard integrations to expand and maintain testing (e2e, jobs) and scale-testing (load, density)
```

Comment:
Which ones?

Reply:
@spiffxp -- can you clarify what "which ones" means? previously I was asked to expand on the scope of what testing and scale testing integration meant hence this line.

Reply:
In terms of jobs this feels too general... are you responsible for any job that happens to run on AWS?

If I read between the lines this sounds like owning the recent eks kubetest deployer, and potentially owning adapters/integrations that allow tools like prow and the perf dashboard to use s3 instead of gcs.. is that the sort of stuff you're trying to imply?

Reply:
@spiffxp as discussed, added the examples in the "out-of-scpe" section and added "on AWS and Amazon EKS" when describing scope of integrations for Prow, Testgrid and Perf Dashboard.

Let me know!